lib/read.js

```diff
@@ -10,12 +11,18 @@ const schema  = require('./schema');
  * @param {String|ReadableStream} file File where the torrent
  *   resides, can be local file, remote, or a readable stream.
  * @param {Function(!Error, Torrent)} callback
+ * @param {Boolean} validate validate or not schema
  * @param {ReadableStream}
  */
-module.exports = (file, callback) => {
+module.exports = (file, validate = true, callback) => {
```

Comment:
no need to have the `= true` part here since this optional param is checked below
```suggestion
module.exports = (file, validate, callback) => {
```